internal/serve/serve.go

```diff
@@ -155,6 +167,47 @@ func initHandlerDeps(cfg Configs) (handlerDeps, error) {
 	}
 	go ensureChannelAccounts(channelAccountService, int64(cfg.NumberOfChannelAccounts))
 
+	// TSS
+	txServiceOpts := tssservices.TransactionServiceOptions{
+		DistributionAccountSignatureClient: cfg.DistributionAccountSignatureClient,
+		ChannelAccountSignatureClient:      cfg.ChannelAccountSignatureClient,
+		HorizonClient:                      &horizonClient,
+		BaseFee:                            int64(cfg.BaseFee), // Reuse horizon base fee for RPC??
```

Comment:
I think that's fair 👍 
```suggestion
		BaseFee:                            int64(cfg.BaseFee),
```

Reply:
Fair